guide/03-the-gis/cloning-content.ipynb

```diff
@@ -0,0 +1,2051 @@
+{
```

Comment:
<p>Can we confirm that <code>nested</code> folder structures are also created in this case?  e.g. a FS exists under folder_level_a\folder_level_b\folder_level_c?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>The folder structure I'm duplicating is the folder structure in the portal app. I don't see how to create nested folders in the app. Do you do that through the REST API? I've not seen that. I can make a note that the code is only one level deep on folders. Let me know what you mean about the nested folders.</p>